template/src/store.ts

```diff
@@ -1,8 +1,10 @@
 import { configureStore } from '@reduxjs/toolkit';
 import counterReducer from './features/counter/counterSlice';
 
-export default configureStore({
+export const store = configureStore({
```

Comment:
I'd actually prefer to make it a named export in both templates.

Reply:
Personally I don't think it makes sense since the file is named store and this is the main export. However, the other file would still need to be updated to fix the import.

Reply:
Right, the import/export syntax needs to be consistent either way.

Overall, I'm just generally leaning away from default exports in most cases in my own code.

Reply:
I dislike default exports because of the poor auto-complete experience, but that's just my opinion.

Reply:
I fixed the import bug though, good catch

Reply:
I prefer default exports but I could go either way, I just wanted to point out that bug

Reply:
@markerikson Do you care about the consistency here? I changed it to a named export but now it's inconsistent with some places where we do default exports

Reply:
Such as https://github.com/reduxjs/cra-template-redux-typescript/pull/3/files#diff-9830f406f48f2a03610b26f834b0692dR58

Reply:
Eh, let's leave `App` as is, and try to go for named exports pretty much everywhere else.